web/src/components/schedule/subcomponents/schedule_timeline/schedule_timeline.js

```diff
@@ -0,0 +1,105 @@
+import { Box, Typography } from "@mui/material";
+import Timeline from "@mui/lab/Timeline";
+import TimelineItem from "@mui/lab/TimelineItem";
+import TimelineSeparator from "@mui/lab/TimelineSeparator";
+import TimelineConnector from "@mui/lab/TimelineConnector";
+import TimelineContent from "@mui/lab/TimelineContent";
+import TimelineDot from "@mui/lab/TimelineDot";
+import TimelineOppositeContent from "@mui/lab/TimelineOppositeContent";
+import scheduleTimelineStyles from "./schedule_timeline.styles.js";
+import { useTheme } from "@emotion/react";
+import messages from "./messages.js";
+
+function ScheduleTimeline() {
+  const theme = useTheme();
+  return (
+    <Box sx={scheduleTimelineStyles.sectionContainer}>
+      <Box sx={scheduleTimelineStyles.section}>
+        <Timeline position="alternate">
```

Comment:
This was the easiest way I could set up info on both sides of the timeline, otherwise the default styling was weird. So I just switched the class names and locations on alternating items

Reply:
fixed